scripts/resume.py

```diff
@@ -133,18 +133,26 @@ def instance_properties(partition, model, placement_group, labels=None):
             "values": [reservation_name],
         }
 
-        placement = util.reservation_placement(reservation)
-        if placement:
-            props.resourcePolicies = [placement]
+        policies = util.reservation_resource_policies(reservation)
+        if policies:
+            props.scheduling = {
+                "onHostMaintenance": "TERMINATE",
+                "automaticRestart": False,
+            }
+            props.resourcePolicies = policies
             log.info(
-                f"reservation {reservation_name} is being used with placement policy {placement}"
+                f"reservation {reservation_name} is being used with policies: {policies}"
             )
         else:
             props.resourcePolicies = []
             log.info(
-                f"reservation {reservation_name} is being used without a placement policy"
+                f"reservation {reservation_name} is being used without any policies"
             )
 
+    if node_group.maintenance_interval:
+        props.scheduling = props.scheduling or {}
+        props.scheduling["maintenanceInterval"] = node_group.maintenance_interval
```

Comment:
Does `null` propagate up to here as `None`? Would it be better to do something like

```python
if node_group.maintenance_interval:
    props.scheduling["maintenanceInterval"] = node_group.maintenance_interval
```

Might be 1-lineable

Reply:
Is either `""`-default or `"PERIODIC"`, I don't expect it to be `null/None`